new
	self error: 'Use #for:'